### DISCUSSION TITLE
Vertical-container does not work after Poseidon Theme Upgrade

### DISCUSSION THREAD

**Discussion by ahmetsaimyilmaz** [primefaces/community#924] [PrimeFaces Templates]
We have upgraded our poseidon theme from 4.0.0 to 6.0.0. I have a panel where items are listed in vertical order using `<div class="p-grid p-align-stretch vertical-container">`. After the upgrade it has no effect and listed horizontally. Looked in the theme.css folder but found no code related to vertical-container. How can I overcome this issue ? Thanks for the help

**Comment by tandraschko** [comment]
vertical-container was a util class in poseidon itself based on old p-grid.
it was later updated to primeflex.

so at best look for "p-grid" classes and replace that all by primeflex
